packages/block-editor/src/components/writing-flow/use-drag-selection.js

```diff
@@ -84,6 +88,16 @@ export default function useDragSelection() {
 					return;
 				}
 
+				// Abort if we are already multi-selecting.
+				if ( isMultiSelecting() ) {
+					return;
```

Comment:
This is the reason we can't continue with the selection right? 

https://github.com/WordPress/gutenberg/assets/16275880/b2e17680-6d39-4144-a830-1bee502ce167